src/scheduler/data_types.h

```diff
@@ -759,6 +759,7 @@ struct resv_info
 	time_t req_start_orig;		/* For standing reservations being modified, this will used to get start time of future occurrences */
 	time_t req_end;			/* user requested end tiem of resv */
 	time_t req_duration;		/* user requested duration of resv */
+	time_t req_duration_orig;	/* For standing reservations being modified, this will used to get duration of future occurrences */
```

Comment:
I see you query this, but don't actually use it.

Reply:
It is used in resv_info.c (line 558)